base.js

```diff
@@ -33,8 +33,8 @@ const KNOWN_WIRE_ADAPTERS = [
         identifier: 'SessionContextAdapter',
     },
     {
-        module: 'commerce/einsteinRecommendationsApi',
```

Comment:
@mlodz-salesforce when does `commerce/einsteinRecommendationsApi` became available for customer? Note: by removing this as a known wire adapter, existing code using `commerce/einsteinRecommendationsApi` will fail to compile.

Reply:
This adapter is being released in 238; it's not yet available to the public, so no worries that it will break existing components.